test/functional/rpc_packages.py

```
@@ -262,6 +275,110 @@
             {"txid": tx1.rehash(), "package-error": "conflict-in-package"},
             {"txid": tx2.rehash(), "package-error": "conflict-in-package"}
         ])
+    def test_rbf(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_rbf`

Add return type annotation: `None`

(ANN201)